minikubek8s.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+
+# Install kubectl
+echo "----------------------------kubectl checking----------------------------"
+export kubectl=$(which kubectl)
+if [ -f "$kubectl" ]; then
+    echo "$kubectl already exist...................................."
+else
+    echo "kubectl does not exist...................................."
+    echo "Installing kubectl........................................"
+    curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/darwin/amd64/kubectl
+    chmod +x ./kubectl
+    sudo mv ./kubectl /usr/local/bin/kubectl
```

Comment:
🤔 I don't know what to think about moving `kubectl` to `/usr/local/bin` as part of this script... 
I don't like scripts moving binaries to my bin folder but, on the other hand, this is a step that probably you will end up doing... 

What do you think about it?

Reply:
i read from kubernetes.io for installation kubectl can use with "brew install kubernetes-cli", but i cannot try in my laptop to makesure kubectl running well